docs/source/parameter_handling.rst

```diff
@@ -39,4 +39,51 @@ The parameters can then be accessed as
     [('param1', 7.0), ('param2', 4.0)]
     >>> p.exists("param1")
     True
+    
+    
+.. code-block:: python
+ 
+    # print all data inside Param
+    def printAll(x):
+      if (x.size()):
```

Comment:
maybe better:
```suggestion
      if not x.empty():
```

if you like the changes you can "Add suggestion to batch" and it will do one commit later to apply the suggestions

Reply:
or use p as suggested above